app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -159,7 +159,10 @@ class BackgroundServices(
             notificationManager.showReceivedTabs(context, device, tabs)
         }
 
-        SyncedTabsIntegration(context, accountManager).launch()
+        SyncedTabsIntegration(
+            syncedTabsStorage = syncedTabsStorage,
```

Comment:
Please don't land this without testing the feature carefully. If I recall correctly this causes a cyclic dependency, which is why we accessed the storage via the context instead of passing it in through he constructor.